README.md

```diff
@@ -28,12 +28,15 @@ end
 
 ## Sandbox API
 
-Uphold supports a sandbox environment for testing purposes. To use it with this strategy, you'll need to use the following environment variables:
+Uphold supports a sandbox environment for testing purposes. To use it with this strategy, you'll need to use a different strategy:
```

Comment:
This sentence does not make a lot of sense, imo. What about "To use it you will need a different strategy"?